ios/brave-ios/Sources/AIChat/Components/Feedback/AIChatFeedbackView.swift

```diff
@@ -0,0 +1,390 @@
+private struct AIChatDropdownView: View {
+  @Binding
+  var selectedItem: AIChatFeedbackOption
+  
+  @State
+  private var showMenu = false
+
+  var body: some View {
+    VStack {
+      HStack(spacing: 0.0) {
+        Text(Strings.AIChat.feedbackOptionsViewTitle)
+          .font(.caption.weight(.semibold))
+          .foregroundStyle(Color(braveSystemName: .textPrimary))
+        
+        Text(verbatim: "*")
+          .font(.caption.weight(.semibold))
+          .foregroundStyle(Color(braveSystemName: .systemfeedbackErrorText))
+      }
+      .frame(maxWidth: .infinity, alignment: .leading)
+      
+      DropdownView(showMenu: $showMenu) {
+        AIChatDropdownButton(action: {
+          showMenu.toggle()
+        }, title: selectedItem.rawValue)
+      } menuView: {
+        AIChatDropdownMenu(selectedItem: $selectedItem, items: AIChatFeedbackOption.allCases)
+          .offset(x: 0.0, y: 12.0)
+          .onChange(of: selectedItem) { _ in
+            showMenu = false
+          }
+      }
+    }
+  }
+}
+
+private struct AIChatFeedbackInputView: View {
+  @ObservedObject
+  var model: AIChatSpeechRecognitionModel
+  
+  @Binding
+  var text: String
+  
+  var body: some View {
+    HStack {
+      ZStack(alignment: .leading) {
+        TextEditor(text: $text)
+          .font(.subheadline)
+          .foregroundStyle(Color(braveSystemName: .textPrimary))
+          .autocorrectionDisabled()
+          .autocapitalization(.none)
+          .submitLabel(.done)
+          .frame(minHeight: 80.0)
+          .osAvailabilityModifiers({ view in
+            if #available(iOS 16.4, *) {
+              view
+                .scrollContentBackground(.hidden)
+            } else {
+              view.introspectTextView { textView in
+                textView.backgroundColor = .clear
+              }
+            }
+          })
+        
+        if text.isEmpty {
+          Text(Strings.AIChat.feedbackInputViewTitle)
+            .font(.subheadline)
+            .foregroundColor(Color(braveSystemName: .textTertiary))
+            .disabled(true)
+            .allowsHitTesting(false)
+            .padding(.vertical, 8.0)
+            .padding(.horizontal, 5.0)
+            .frame(maxHeight: .infinity, alignment: .top)
+        }
+      }
+      .fixedSize(horizontal: false, vertical: true)
+      .padding(.trailing)
+      
+      if model.speechRecognizer.isVoiceSearchAvailable {
+        Button {
+          Task { @MainActor in
+            let permissionStatus = await model.speechRecognizer.askForUserPermission()
+            if permissionStatus {
+              model.isVoiceEntryPresented = true
+              model.activeInputView = .feedbackView
+            } else {
+              model.isNoMicrophonePermissionPresented = true
+              model.activeInputView = .none
+            }
+          }
+        } label: {
+          Image(braveSystemName: "leo.microphone")
+            .foregroundStyle(Color(braveSystemName: .iconDefault))
```

Comment:
Switch to a `Label` + `iconOnly` `LabelStyle` like xmark to provide a VoiceOver string